src/pages/zoeken/[slug]/[ie]/index.tsx

```diff
@@ -342,10 +342,15 @@ const ObjectDetailPage: NextPage<ObjectDetailPageProps> = ({ title, description,
 
 	useEffect(() => {
 		if (mediaInfo) {
-			EventsService.triggerEvent(LogEventType.ITEM_VIEW, window.location.href, {
+			const path = window.location.href;
+			const eventData = {
 				schema_identifier: mediaInfo.schemaIdentifier,
 				meemoo_identifier: mediaInfo.meemooIdentifier,
-			});
+				user_group_name: user?.groupName,
+			};
+
+			EventsService.triggerEvent(LogEventType.BEZOEK_ITEM_VIEW, path, eventData);
```

Comment:
> Mogen blijven bestaan (deze worden enkel uitgestuurd als het gaat om een sessie waar bezoekertool actief is, en de eindgebruiker een item binnen de bezoekertool zoekt of raadpleegt):

So we should wrap this Visit event inside a mediaInfo.accessThrough.includes('VISIT...')?